docs/user_guide.md

```diff
@@ -335,14 +360,48 @@ module "forseti" {
       project_id         = "your-forseti-project-id-here"
       org_id             = "your-org-id-here"
       …
-      config_validator_enabled = true
+      config_validator_enabled      = true
+      policy_library_sync_enabled   = true
+      policy_library_repository_url = "git@github.com:forseti-security/policy-library"
     }
 ```
 
-The one important additions is the `config_validator_enabled` field. It is not
-enabled by default; therefore you need to explicitly enable it.
+The important additions are the `config_validator_enabled` and
+`policy_library_sync_enabled` fields. They are not enabled by default; therefore
+you need to explicitly enable them. It is recommended to use the [Policy Library Sync](#policy-library-sync) feature, but not required. If you are using the
+sync, then set `policy_library_repository_url` to the Git URL of your private
+repo.
+
+### Policy Library Sync
+
+Forseti has the ability to clone your private policy library repo and keep it in
+sync (using [git-sync](https://github.com/kubernetes/git-sync)), so that config
+validator will always use the latest policies while scanning. Optionally, you
+can provide the SSH known hosts to use when Forseti clones/pulls the repo. To
+provide the known hosts, run the command below and use the output as the value
+for the `policy_library_sync_ssh_known_hosts` field within the main.tf:
+
+```
+ssh-keyscan ${YOUR_GIT_HOST}
+```
+
+After applying the Terraform configuration, you will need to add the generated
+SSH key to the read-only user's account. The SSH key will be provided as an
+output from Terraform. If your private repo is hosted on GitHub, you can
+[follow these steps to add the SSH key to your account](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account).
```

Comment:
(non blocking) Instead of adding the ssh key directly to the user's account, is there any way to make it read-only access? Assuming the user is the admin of a few repos, if you were to add the SSH key to the profile directly, I think you can update/delete all the repos under that user?

Reply:
The only way to make it read-only is to setup a read-only user within the organization, which is recommended to be done in the [setup section](https://github.com/gkowalski-google/policy-library/blob/feature/policy-library-sync-documentation/docs/user_guide.md#how-to-set-up-constraints-with-policy-library).